public/app/plugins/panel/histogram/Histogram.tsx

```diff
@@ -204,6 +204,13 @@ const prepConfig = (frame: DataFrame, theme: GrafanaTheme2) => {
       y: false,
       setScale: true,
     },
+    dataIdx: (u, _, closestIdx, xValue) => {
+      let xTiker = u.data[0][closestIdx];
+      if (xValue < xTiker) {
```

Comment:
variable name is a bit odd, let's just inline the condition?

```suggestion
      if (xValue < u.data[0][closestIdx]) {
```

Reply:
heh, i get it (i wrote uPlot). the code makes sense.

i was just commenting in the variable name since you said the name was derived from "tick" :)